### PR TITLE
v9 update

### PR DESCRIPTION
**Pull request by jessev14**
small changes to fix entity -> document related bugs

### CODE REVIEW

File: src/controller.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export function delayCallback(callback: (...args: unknown[]) => boolean, ...args
     if (timers[callback.name])
         window.clearTimeout(timers[callback.name]);
 
-    timers[callback.name] = window.setTimeout(() => callback(...args), 100);
+    timers[callback.name] = window.setTimeout(() => callback(...args), 500);
```
Comment (Varriount) [inline src/controller.ts:63]: Why was this changed?

Comment (jessev14) [reply src/controller.ts:63]: I was running into some error during initialization when the delay was 100ms, but it seems to be gone now. should be able to ignore this change.

**Review by Varriount — APPROVED** [review]
